topdown/http.go

```diff
@@ -568,3 +622,349 @@ func parseTimeout(timeoutVal ast.Value) (time.Duration, error) {
 		return timeout, builtins.NewOperandErr(1, "'timeout' must be one of {string, number} but got %s", ast.TypeName(t))
 	}
 }
+
+func getBoolValFromReqObj(req ast.Object, key string) (bool, error) {
```

Comment:
nit: it's odd that this function is called `getBoolValFromReqObj` when really it's just accepting an object and a string and returning a boolean (or an error if the value is not a boolean). We could refactor this function to avoid that, but it's almost unnecessary if other code is reorganized. Also, avoid converting values to strings unnecessarily.

```golang
var b ast.Boolean

/* key can be constructed once above as a term for force_json_decode, etc. */
if v := obj.Get(key); v != nil {
  if b, ok = v.Value.(ast.Boolean); !ok {
     // error
  }
}

Reply:
Updated.